data/mods/PKs_Rebalance/pk_critter_doom.json

```diff
@@ -439,7 +430,7 @@
             "move_cost" : 95,
             "targeting_cost" : 300,
             "range" : 21,
-            "description" : "As the caccodemon opens its mouth a torrent of plasma bursts forth!",
+            "description" :"As the cacodemon opens its mouth a torrent of plasma bursts forth!",
```

Comment:
It's not such a big deal, but anyways. In this PR all four possible combinations of spaces and colons were used.
```
"key" :"value"
"key": "value"
"key" : "value"
"key":"value"
```